include/GafferScene/SceneAlgo.h

```diff
@@ -84,6 +84,11 @@ GAFFERSCENE_API void matchingPaths( const Gaffer::IntPlug *filterPlug, const Sce
 /// As above, but specifying the filter as a PathMatcher.
 GAFFERSCENE_API void matchingPaths( const IECore::PathMatcher &filter, const ScenePlug *scene, IECore::PathMatcher &paths );
 
+/// Matching above, but doing a fast hash of the matching paths instead of storing all paths
+GAFFERSCENE_API IECore::MurmurHash matchingPathsHash( const Filter *filter, const ScenePlug *scene );
+GAFFERSCENE_API IECore::MurmurHash matchingPathsHash( const Gaffer::IntPlug *filterPlug, const ScenePlug *scene );
```

Comment:
We should take  `const FilterPlug *` here instead of `IntPlug`. The `matchingPaths( IntPlug )` form is only for back-compatibility, and has been updated on master. And that probably explains the CI errors, since this PR is being tested against master :

```
src/GafferScene/SceneAlgo.cpp:180:73: error: no matching function for call to 'filteredParallelTraverse(const GafferScene::ScenePlug*&, const IntPlug*&, {anonymous}::ThreadablePathHashAccumulator&)'
  GafferScene::SceneAlgo::filteredParallelTraverse( scene, filterPlug, f );
```

Reply:
Fixed.